### PR TITLE
Created Travis continious integration setup

### PR DESCRIPTION
**Pull request by davetcoleman**
Adding continuous integration to all ros-controls repos for better pull request testing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - cpp
+  - python
+python:
+  - "2.7"
+compiler:
+  - gcc
+notifications:
+  email:
+    recipients:
+      - davetcoleman@gmail.com
```
Comment (adolfo-rt) [inline .travis.yml:13]: Mind if I add my email here as well?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - rosdep update
+install: # Use this to install any prerequisites or dependencies necessary to run your build
+  # Create workspace
+  - mkdir -p ~/ros/ws_catkin/src
+  - cd ~/ros/ws_ros_controls/src
+  - wstool init .
+  # Download non-debian stuff
+  - wstool merge https://raw.github.com/ros-controls/ros_control/hydro-devel/ros_control.rosinstall
+  - wstool update
+  # Delete the moveit.rosinstall version of this repo and use the one of the branch we are testing
```
Comment (adolfo-rt) [inline .travis.yml:37]: This moveit-specific comment slipped by.

**Comment by adolfo-rt** [comment]
Nice!. Validating pull requests will be a big plus. Integration is not yet fully tested, right?

File: .travis.yml (hunk 1 above)
Comment (davetcoleman) [inline .travis.yml:13]: Please do! I don't want to be the only one getting these notifications

File: .travis.yml (hunk 2 above)
Comment (davetcoleman) [inline .travis.yml:37]: Fixed, thanks!

**Comment by davetcoleman** [comment]
Correct, I haven't worked out all the kinks yet

**Pull request merged**